src/sentry/models/projectkey.py

```diff
@@ -36,8 +36,8 @@ class ProjectKeyStatus(object):
 class ProjectKey(Model):
     project = FlexibleForeignKey('sentry.Project', related_name='key_set')
     label = models.CharField(max_length=64, blank=True, null=True)
-    public_key = models.CharField(max_length=32, unique=True, null=True)
-    secret_key = models.CharField(max_length=32, unique=True, null=True)
+    public_key = EncryptedCharField(max_length=32, unique=True, null=True)
+    secret_key = EncryptedCharField(max_length=32, unique=True, null=True)
```

Comment:
I'd probably say these Encrypted fields should just be `TextField` based unless the length can reliably be determined.

Either way, to Postgres at least, a `char` type is the same as `text` just with a constraint, so I think we can live without a length constraint here imo.

We'd also likely need to move this `unique` indexes to the new hashed fields since I think we determined that even identical values should be encrypted into different values, so uniqueness could only apply to the hash.